addons/nvidia/0.3.x/nvidia-1.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: nvidia
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: nvidia
+    kubeaddons.mesosphere.io/provides: nvidia
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.2.0-1"
```

Comment:
If the appversion is 0.2.x, should we not reflect that in the directory? Currently it's under `0.3.x` which is the chart version (but not the app version I take it)?

Reply:
good catch, thanks!